tests/lms/views/decorators/test_h_api.py

```diff
@@ -111,42 +111,25 @@ def create_h_user(self, wrapped):
 
 @pytest.mark.usefixtures("hapi_svc")
 class TestCreateCourseGroup:
-    @pytest.mark.parametrize(
-        "required_param_name", ("tool_consumer_instance_guid", "context_id", "roles")
-    )
-    def test_it_400s_if_theres_a_required_param_missing(
-        self, create_course_group, context, pyramid_request, required_param_name
+    def test_it_400s_if_roles_param_missing(
```

Comment:
Can you help me understand why only `roles` is required now? I know that `context_id` is used to generate a `groupid`, but does this now happen elsewhere such that this and other params aren't required _here_ anymore?